dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -1196,6 +1197,11 @@ public long count(User user, boolean respectFrontendRoles) {
         }
     }
 
+    @VisibleForTesting
+    public void flushCache(Host host){
```

Comment:
I think we should create something such as 

```
interface Flushable<T> {

    void flushAll();
    void flush(T t);
}
```

Then you can use

Flushable.class.cast(ApiLocator.getHostAPI())).flushCache(host)

Reply:
done https://github.com/dotCMS/core/pull/20293/commits/24854ac1172f4bf79943d66ba236d339d981052a#diff-5b9be74951ac077907286368b9f40cf1133b6942e3e6e04bba11eac6c5912896R1